ci-operator/config/openshift/kube-proxy/openshift-kube-proxy-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.1"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+canonical_go_repository: github.com/openshift/origin
```

Comment:
Huh, if this needs to be built from `openshift/origin`, why is this in `openshift/kube-proxy`?

Reply:
@petr-muller OK, I need help here. kube-proxy is hyperkube with 2 additional rpms. We build it in ose-kube-proxy for OCP and need to do the equivalent in origin. What do I need to do to get this to happen?

Reply:
I do not know enough about what you are trying to achieve, but if this is an image that should be built from `openshift/origin`, then I would expect the image to be added to the `openshift/origin` configs in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin. This would cause these images to be built both before (to test) and after (to promote) any PR to `openshift/origin`.